src/ua/lviv/iot/algo/part1/lab1/Plate.java

```diff
@@ -0,0 +1,47 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+
+public class Plate {
+
+    private static Plate defaultPlate = new Plate();
```

Comment:
Since it is a pattern, it is better to use a common name "instance". defaultPlate is a plate with default values out of singleton context